test/unit/services/data-test.js

```diff
@@ -106,6 +107,18 @@ describe('InstallerDataService', function() {
     });
   });
 
+  describe('copyUninstaller', function(){
+    it('should be able to handle copyUninstaller', function(){
+      let data = {
+          darwin: ()=>'darwin',
+      };
+      sandbox.stub(Platform,'getOS').returns('darwin');
+      expect(Platform.identify(data)).to.be.equal('darwin');
```

Comment:
You should remove lines 112-114 and 116, because they do not needed here. Line 115 is enough to make installer thing it is running on macOS.